tests/phpunit/payment-methods/test-wc-stripe-express-checkout-helper.php

```diff
@@ -311,4 +311,16 @@ public function provide_test_get_normalized_postal_code() {
 			],
 		];
 	}
+
+	/**
+	 * Test for `get_payment_method_title_suffix`
+	 *
+	 * @return void
+	 */
+	public function test_get_payment_method_title_suffix() {
+		$wc_stripe_ece_helper = new WC_Stripe_Express_Checkout_Helper();
+		$actual               = $wc_stripe_ece_helper->get_payment_method_title_suffix();
```

Comment:
Since `get_payment_method_title_suffix` is a static method, instantiating the class is unnecessary. 

```suggestion
		$actual = WC_Stripe_Express_Checkout_Helper::get_payment_method_title_suffix();
```